### DISCUSSION TITLE
How to config custom app_name etc?

### DISCUSSION THREAD

**Discussion by JuanAlfonsus** [LeonardSSH/vscord#221] [General]
I dont really understand how to edit Configuration.
Do I need to make my own Custom Discord Rich Pressence by cloning it and edit config.ts?

**Comment by ijustgotanidea** [comment]
Same. Didnt understand at all how to edit my configuration.

**Comment by PunGrumpy** [comment]
On `app_name` or `vscord.app.name`. If you want to change it to your own name, you should select **'Custom'** and then replace `vscord.app.id` with your own Application's ID from the [Discord development](https://discord.com/developers/applications) portal.

**Comment by TinnyTerr** [comment]
I did this a while ago and these are the steps I did to do such:
1. Go to [discord.dev](https://discord.com/developers/applications) and log in if needed.
2. Create a App and name it whatever (some times requires the name of the App to be the Rich Presence title)
3. Copy the client/app ID
4. Paste into the extension settings in App ID 
5. Name the App whatever the title will be or in vscode name it the title you want it.
6. Should then appear after a reload of the extention or vscode
